tests/interfaces/signal_test.py

```diff
@@ -0,0 +1,64 @@
+import arrow
+import mock
+import pytest
+import staticconf
+from clusterman_metrics import APP_METRICS
+from clusterman_metrics import METADATA
+from clusterman_metrics import SYSTEM_METRICS
+
+from clusterman.exceptions import MetricsError
+from clusterman.interfaces.signal import get_metrics_for_signal
+
+
+@pytest.mark.parametrize('end_time', [arrow.get(3600), arrow.get(10000), arrow.get(35000)])
+def test_get_metrics(end_time):
+
+    required_metrics = staticconf.read_list(
+        'autoscale_signal.required_metrics',
+        namespace='bar.mesos_config',
+        default=[],
```

Comment:
nit: I'd remove this so that in the event the pool config is somehow removed, we'll see an error with reading the list, instead of an error with the assert lower down.

Reply:
O, I specifically meant removing the `default=[]` line, not the entirety of what was highlighted, that's my bad.